### DISCUSSION TITLE
Newline in specific directories waits for ctrl-c?

### DISCUSSION THREAD

**Discussion by ramyadhadidi** [ohmyzsh/ohmyzsh#11698] [Help]
Hello,

I have a strange issue and I am not even sure how it happens. When I `cd` to specific directories, the prompt waits for newlines and mostly never stops until I do ctrl-c. This doesn't happen when I am on with bash. I use iTerm2, but the same issue also happens when I use the default Mac Terminal app. All the machines that this happens are Ubuntu based.

**[Update]**
It is related to the themes. For instance, `cypher` does not have this issue.

![Screenshot 2023-05-18 at 07 23 37](https://github.com/ohmyzsh/ohmyzsh/assets/5882631/520451b5-be99-416b-992e-b06b76f6cf38)

- Tested with a fresh ohmyzsh installation, still happens.
- Tested with no Git plugin (and no other plugins), still happens.
- Changed a few themes, still happens.

**Comment by rwmitchell** [comment]
You said add specific directories, not all?  Can you determine anything they have in common?

**Reply by ramyadhadidi** [reply]
Nothings specific in their names that stand out. Also, nothing similar in the files.

**Reply by rwmitchell** [reply]
Git repo?

**Reply by ramyadhadidi** [reply]
I disabled the git plugin it those directories. All are git repos. And still happens.

I changed a few themes. And among them only `cypher` does not have this issue. So, it is an issue with the theme, not git plugin?

**Comment by mcornella** [comment]
You can enable zsh trace mode with `set -xv` and then press Enter to see what functions / zsh code is running while you are waiting. Note that this will output a lot of text, but we are only interested in the last parts (to see if it hangs at a particular line of code, or what code is running in particular).

It would be good also to know with which particular themes this happens.
